Tests/Locale/DefaultProviderTest.php

```diff
@@ -43,7 +43,7 @@ public function testDefaultLocaleIsInLocales()
 
         // set expectations for constructor calls
         $this->setExpectedException(
-            'InvalidArgumentException', 'Default locale should be contained in locales'
+            'InvalidArgumentException', 'Default locale `de` should be contained in locales'
```

Comment:
Why are you changing this line?

Reply:
Normally it should't be in the PR. it was only to check if the correct test was used, as I had a doubt about it.
Only change needed for the fix was in services.xml

Reply:
Revert this change then